jenkins.yaml

```diff
@@ -271,23 +235,23 @@ unclassified:
               - checkoutOption:
                   timeout: 60
               - userIdentity:
-                  email: "customuser@acmecorp.com"
+                  email: "customuser@example.com"
                   name: "custom user"
               - preBuildMerge:
                   options:
                     mergeRemote: "myrepo"
                     mergeStrategy: RECURSIVE
                     mergeTarget: "master"
               gitTool: "Default"
-              submoduleCfg:
-              - submoduleName: "submodule-1"
-                branches:
-                - "mybranch-1"
-                - "mybranch-2"
-              - submoduleName: "submodule-2"
-                branches:
-                - "mybranch-3"
-                - "mybranch-4"
+              # submoduleCfg:
```

Comment:
Why? it works with the current plugin versions

Reply:
Didn't work in the environment that I was using.  I suspect the outdated plugin from CAP was being used instead of the version described in plugins.txt